ruby/vampires.rb

```diff
@@ -0,0 +1,66 @@
+# Vampire Tester
+
+applicant_status = nil
+current_year = 2017
+allergy = nil
+
+puts "How many employees are we processing today?"
+number_of_applicants = gets.chomp.to_i
+
+while number_of_applicants > 0
+
+  puts "What is your name?"
+  applicant_name = gets.chomp
+
+  puts "How many years old are you?"
+  years_old = gets.chomp.to_i
+
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+
+  puts "Our cafeteria serves garlic bread. Should we order some for you? (Yes or No)"
+  garlic_bread = gets.chomp.downcase
+
+  puts "Would you like to enroll in the company's health insurance? (Yes or No)"
+  insurance = gets.chomp.downcase
+
+  until allergy == "sunshine" || allergy == "done"
+    puts "Please name any allergy you have. Type 'done' if you don't have any or if you're finished listing allergies."
+    allergy = gets.chomp
+    if allergy == "done"
+      puts "Good to know. We can continue with the application."
+    end
+  end
+  
+  if current_year - birth_year == years_old
+    correct_year = true
+  else
+    correct_year = false
+  end
+
+  
+  if correct_year && (garlic_bread == "yes" || insurance == "yes")

```

Comment:
Just like you turned `correct_year` into a boolean value for the conditional statements, you can do the same with garlic and health insurance. For instance the long form of this could be:
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
and that can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.